include/swift/Option/Options.td

```diff
@@ -247,6 +247,15 @@ def disable_swift_bridge_attr : Flag<["-"], "disable-swift-bridge-attr">,
   Flags<[FrontendOption, HelpHidden]>,
   HelpText<"Disable using the swift bridge attribute">;
 
+def enable_swift3_objc_inference : Flag<["-"], "enable-swift3-objc-inference">,
+  Flags<[FrontendOption, HelpHidden]>,
+HelpText<"Enable Swift 3's @objc inference rules for NSObject-derived classes and 'dynamic' members (emulates Swift 3 behavior)">;
+
+def disable_swift3_objc_inference :
```

Comment:
You made these driver options, but forgot to forward them in `addCommonFrontendArgs`. (If they're supposed to *just* be frontend options, please hide them in FrontendOptions.td.)